src/components/refactor/dialogs/generator/reactive-limits/reactive-capability-curve/reactive-capability-curve-table.js

```diff
@@ -15,16 +15,46 @@ import { useStyles } from '../../../../../dialogs/dialogUtils';
 import { useFieldArray } from 'react-hook-form';
 import ReactiveCapabilityCurveRowForm from './reactive-capability-curve-row-form';
 import ErrorInput from '../../../../rhf-inputs/error-inputs/error-input';
+import {
+    OLD_P,
+    OLD_Q_MAX_P,
+    OLD_Q_MIN_P,
+    P,
+    Q_MAX_P,
+    Q_MIN_P,
+} from 'components/refactor/utils/field-constants';
 import MidFormError from 'components/refactor/rhf-inputs/error-inputs/mid-form-error';
+import { INSERT, REMOVE } from './reactive-capability-utils';
 
 export const ReactiveCapabilityCurveTable = ({
     id,
     tableHeadersIds,
     disabled = false,
+    previousValues,
+    updateReactiveCapabilityCurveTableRow,
 }) => {
     const { fields: rows, insert, remove } = useFieldArray({ name: `${id}` });
     const classes = useStyles();
 
+    const handleInsertRow = () => {
+        if (previousValues && updateReactiveCapabilityCurveTableRow)
+            updateReactiveCapabilityCurveTableRow(INSERT, rows.length - 1);
```

Comment:
I think it would be more understandable to rename it to "updatePreviousReactiveCapabilityCurveTable"

Reply:
well i renamed